packages/web3torrent/src/context/channel-context.ts

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Too many context classes. Don't see any async calls here. Is this class necessary? 😛